server/persistence/persistence_test.go

```diff
@@ -492,3 +493,175 @@ func testSaveGameWithMissingAction(t *testing.T, name dbName, db persistence.DB)
 		assert.Error(t, db.SaveGame(g), `saving a game with a corrupted action is a :badtime:`)
 	}
 }
+
+func TestTransactionality(t *testing.T) {
+	// Right now (and probably ever) the memory persistence isn't transactional
+	dbs := map[string]func() persistence.DB{
+		// `memory`: func() persistence.DB { return inmem },
+	}
+
+	if !testing.Short() {
+		dbs[`mongodb`] = func() persistence.DB {
+			// We assume you have mongodb stood up locally when running without -short
+			// we change the uri because travis sets up a different mongodb replica set than run-rs does
+			mongo, err := mongodb.New(context.Background(), `mongodb://127.0.0.1:27017,127.0.0.1:27018/?replicaSet=testReplSet`)
```

Comment:
Can't find anything on this online, but did you make a new replica set called `testReplSet`? When I spin up mongo using `run-rs` it defaults to being called `rs` with no apparent way to change it.

If I change `testReplSet` to `rs` in the URI, the test proceeds...

Reply:
TBH, I haven't run the unit tests locally in a while. I'm gonna be content with it as is:#